x-pack/solutions/security/plugins/elastic_assistant/server/lib/attack_discovery/schedules/fields/field_names.ts

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { ALERT_NAMESPACE } from '@kbn/rule-data-utils';
+
+export const ALERT_ATTACK_DISCOVERY = `${ALERT_NAMESPACE}.attack_discovery` as const;
+
+export const ALERT_ATTACK_DISCOVERY_TITLE = `${ALERT_ATTACK_DISCOVERY}.title` as const;
```

Comment:
nit: Consider ordering the entries in this file alphabetically

Reply:
done in https://github.com/elastic/kibana/pull/215184/commits/05cc112bd7260691402f301864bacdc2cf1d340b